db/migrations/20230922154941_add_config_constraint.up.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+
+DO $$
+BEGIN
+
+BEGIN
+alter table repository_configurations ADD CONSTRAINT repo_uuid_and_org_id_del_at_unique UNIQUE (repository_uuid, org_id, deleted_at);
+EXCEPTION
+    WHEN duplicate_table THEN  -- postgres raises duplicate_table at surprising times. Ex.: for UNIQUE constraints.
+    WHEN duplicate_object THEN
+      RAISE NOTICE 'Table constraint repo_uuid_and_org_id_del_at_unique already exists';
+END;
+
+END $$;
```

Comment:
This was the only way i could see to check if a constraint already existed to make this idempotent

Reply:
what about dropping the constraint before adding it?

Reply:
This constraint may actually not be needed, the test failure is related to the unique index for the same fields.  anyway i'm playing around with it

Reply:
yep, it wasn't needed at all.  removed!